nexus/external-api/src/lib.rs

```diff
@@ -3542,49 +3542,116 @@ pub trait NexusExternalApi {
         params: TypedBody<params::DeviceAccessTokenRequest>,
     ) -> Result<Response<Body>, HttpError>;
 
-    // Webhooks
+    // Alerts
 
-    /// List webhook event classes
+    /// List alert classes
     #[endpoint {
         method = GET,
-        path = "/v1/webhooks/event-classes",
-        tags = ["system/webhooks"],
+        path = "/v1/alert-classes",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_event_class_list(
+    async fn alert_class_list(
         rqctx: RequestContext<Self::Context>,
         pag_params: Query<
-            PaginationParams<EmptyScanParams, params::EventClassPage>,
+            PaginationParams<EmptyScanParams, params::AlertClassPage>,
         >,
-        filter: Query<params::EventClassFilter>,
-    ) -> Result<HttpResponseOk<ResultsPage<views::EventClass>>, HttpError>;
+        filter: Query<params::AlertClassFilter>,
+    ) -> Result<HttpResponseOk<ResultsPage<views::AlertClass>>, HttpError>;
 
-    /// List webhook receivers
+    /// List alert receivers
     #[endpoint {
         method = GET,
-        path = "/v1/webhooks/receivers",
-        tags = ["system/webhooks"],
+        path = "/v1/alert-receivers",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_list(
+    async fn alert_receiver_list(
         rqctx: RequestContext<Self::Context>,
         query_params: Query<PaginatedByNameOrId>,
-    ) -> Result<HttpResponseOk<ResultsPage<views::WebhookReceiver>>, HttpError>;
+    ) -> Result<HttpResponseOk<ResultsPage<views::AlertReceiver>>, HttpError>;
 
-    /// Fetch webhook receiver
+    /// Fetch alert receiver
     #[endpoint {
         method = GET,
-        path = "/v1/webhooks/receivers/{receiver}",
-        tags = ["system/webhooks"],
+        path = "/v1/alert-receivers/{receiver}",
+        tags = ["system/alerts"],
     }]
-    async fn webhook_receiver_view(
+    async fn alert_receiver_view(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::WebhookReceiverSelector>,
-    ) -> Result<HttpResponseOk<views::WebhookReceiver>, HttpError>;
+        path_params: Path<params::AlertReceiverSelector>,
+    ) -> Result<HttpResponseOk<views::AlertReceiver>, HttpError>;
+
+    /// Delete alert receiver
+    #[endpoint {
+            method = DELETE,
+            path = "/v1/alert-receivers/{receiver}",
+            tags = ["system/alerts"],
+        }]
```

Comment:
Interesting choices by rustfmt here, huh

Reply:
oh that might actually be my fault, i don't know if rustfmt will format inside attributes like this...